webpack/assets/javascripts/react_app/redux/consts.js

```diff
@@ -0,0 +1,3 @@
+export const STATISTICS_DATA_REQUEST = 'STATISTICS_DATA_REQUEST';
+export const STATISTICS_DATA_SUCCESS = 'STATISTICS_DATA_SUCCESS';
```

Comment:
this is more common approach for action names...
that way you can explicitly import the action names you need, which makes the code more readable as the reader sees which actions the code will use